src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -31,18 +15,9 @@ Append the default ASP.NET Core runtime package store manifest during publish
         Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))'">osx.10.12-x64</ManifestRuntimeIdentifier>
       <ManifestRuntimeIdentifier
         Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))'">linux-x64</ManifestRuntimeIdentifier>
-    </PropertyGroup>
-
-    <Error
```

Comment:
What happens if we leave the target around (but leave the resolution globally)? Can we get an error message in that case?